### ISSUE TITLE
Missing "CompanyCurrency" object

### ISSUE THREAD

**Issue by arik-dokka** [sidecars/python-quickbooks#135]
Greetings. Any plans to add this object?

**Comment by ej2** [comment]
Resolved in 0.8 release.

**Comment by antetna** [comment]
What am I doing wrong with this because other objects are working just fine but not this one CompanyCurrency

```
from quickbooks import QuickBooks
client = QuickBooks(
         auth_client=auth_client,
         refresh_token=REFRESH_TOKEN,
         company_id=REALM_ID,
     )
from quickbooks.objects.companycurrency import CompanyCurrency
cc = CompanyCurrency().all(qb=client)
```

```
Traceback (most recent call last):
  File "/home/user/myprojects/quickbooks/main.py", line 35, in <module>
    cc = CompanyCurrency().all(qb=client)
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 184, in all
    max_results=max_results, qb=qb)
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 235, in where
    return cls.query(select, qb=qb)
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 253, in query
    obj_list.append(cls.from_json(item_json))
  File "/home/user/myprojects/quickbooks/venv/lib/python3.6/site-packages/quickbooks/mixins.py", line 31, in from_json
    sub_obj = sub_obj.from_json(json_data[key])
AttributeError: 'MetaData' object has no attribute 'from_json'
```
